x/emissions/proto/emissions/v5/query.proto

```diff
@@ -409,6 +409,21 @@ service QueryService {
     option (cosmos.query.v1.module_query_safe) = true;
     option (google.api.http).get = "/emissions/v5/count_forecaster_inclusions_in_topic/{topic_id}/{forecaster}";
   }
+
+  rpc GetActiveReputersForTopic(GetActiveReputersForTopicRequest) returns (GetActiveReputersForTopicResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/emissions/v4/active_reputers/{topic_id}";
```

Comment:
should these say v5 in the url link? should we perhaps set "vX" to a variable in the proto that's easily updateable so errors like this don't happen in the future?

Reply:
good catch!